scripts/_aws.sh

```diff
@@ -25,6 +25,8 @@ function _aws() {
 }
 
 function _aws_login() {
+    _aws_reset_creds
```

Comment:
Cheeky addition here

Reply:
how does that sit with you, i'm sick of forgetting to reset creds before login